walking:
  velocity: 0.1
  acceleration: 0.8
  duty_factor: 0.8
  stride_time: 1.5
  stride_height: 0.15
  relative_phase: [0, 0.5, 0.8, 0.3]